public/static/docs/use-cases/data-registries.md

```diff
@@ -109,7 +109,8 @@ This downloads `music/songs/` from the <abbr>project</abbr>'s
 current working directory (anywhere in the file system with user write access).
 
 > Note that this command (as well as `dvc import`) has a `--rev` option to
-> download specific versions of the data.
+> download the data from a specific
+> [revision](https://git-scm.com/docs/revisions) of the source project.
```

Comment:
let's use commit here ? we can keep even the link, but use simple language

Reply:
OK, changed throughout that doc in a0c51ff1. I thought the word "revision" was appropriate since the option is `--rev` but this will hopefully be changed in core repo to more "commit" as well.

Reply:
commit with a link should be enough, when we want to be extra clear we can do `commit (revision[link])